packages/os/os.spec

```diff
@@ -197,7 +211,6 @@ Summary: Dynamic setting generator for kubernetes
 
 %package -n %{_cross_os}static-pods
 Summary: Manages user-defined K8S static pods
-Requires: %{_cross_os}apiserver = %{version}-%{release}
 %description -n %{_cross_os}static-pods
 %{summary}.
 %endif
```

Comment:
Can't comment on line 220... rebase pulled in bootstrap-containers which breaks the pattern.

Reply:
>Requires: %{_cross_os}bootstrap-containers

This should move from `release.spec` to the main package in `os.spec`, as part of our co-authored commit.

Reply:
[fixed](https://github.com/bottlerocket-os/bottlerocket/compare/5fcc3cf03741eae766490d027398585e21a5701f..4477e171823e0947f5b085a6ccf795c337832b99)